hibernate-core/src/main/java/org/hibernate/query/sqm/mutation/internal/SqmMultiTableMutationStrategyProviderStandard.java

```diff
@@ -32,7 +32,11 @@ public SqmMultiTableMutationStrategy createMutationStrategy(
 		if ( specifiedStrategy != null ) {
 			return specifiedStrategy;
 		}
-
+		final SqmMultiTableMutationStrategy specifiedEntityBaseStrategy =
+				options.resolveCustomSqmMultiTableMutationStrategy( rootEntityDescriptor, creationProcess );
+		if ( specifiedEntityBaseStrategy != null ) {
+			return specifiedEntityBaseStrategy;
+		}
```

Comment:
Shouldn't this be _before_ the call to `getCustomSqmMultiTableMutationStrategy()`? It seems to me that if `resolveCustomSqmMultiTableMutationStrategy` returns a non-null value, `getCustomSqmMultiTableMutationStrategy` necessarily will as well, because they are based on the same configuration property... ?

Reply:
I changed the implementation so that `getCustomSqmMultiTableMutationStrategy()` will return `null` if the strategy class has a constructor that accepts `rootEntityDescriptor, creationProcess`.